### PR TITLE
DOC: updated origin param of scipy.ndimage.filters.convolve

### LINKED ISSUE #4580: scipy.ndimage.filters.convolve documenation is incorrect
the meaning of the origin parameter is different:
default value of 0 is equivalent to the center of the filter, and the origin parameter is counted from the center, not from weights[0,0].

#!/usr/bin/env python

import numpy as np
import scipy.ndimage as nd

a = np.zeros([11,11])
a[5,5] = 1.0

b = np.ones([5,5])

print "Without origin: "
print nd.convolve( a, b, mode='constant', cval=0, )

print"With origin (0,0): "
print nd.convolve( a, b, mode='constant', cval=0, origin=(0,0) )

print "With origin (1,1): "
print nd.convolve( a, b, mode='constant', cval=0, origin=(1,1) )

print "With origin (-1,-1): "
print nd.convolve( a, b, mode='constant', cval=0, origin=(-1,-1) )

### PR DESCRIPTION
**Pull request by pizzathief**
fixes #4580 

Seems a stated default origin of (0,0) makes more sense than "0" .  Should I change that as well?  (Should the function even accept origin=0 as an argument?)

### CODE REVIEW

**Comment by WarrenWeckesser** [comment]
This is the **nd**image module.  `convolve` can be applied to n-dimensional arrays, so the default origin is really `(0,)*input.ndim`.

**Comment by pizzathief** [comment]
On 19/10/15 12:17, Warren Weckesser wrote:
>
> This is the *nd*image module. |convolve| can be applied to 
> n-dimensional arrays, so the default origin is really |(0,)*input.ndim|.
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/scipy/scipy/pull/5389#issuecomment-149072624>.
>

right, n-dimensional tuples, as opposed to integers.

File: scipy/ndimage/filters.py [hunk 1]
```diff
@@ -607,8 +607,9 @@ def convolve(input, weights, output=None, mode='reflect', cval=0.0,
         Value to fill past edges of input if `mode` is 'constant'. Default
         is 0.0
     origin : array_like, optional
-        The `origin` parameter controls the placement of the filter.
-        Default is 0.
+        The `origin` parameter controls the placement of the filter, 
+        relative to the centre of the current element of the input.  
+        Default of 0 is equivalent to (0,0).
```
Comment (ev-br) [inline scipy/ndimage/filters.py]: This could be merged once this line is fixed as per Warren's suggestion.

**Comment by ev-br** [comment]
Squashed and merged in https://github.com/scipy/scipy/commit/49265545d704391eae974c35d6c1d5258b1845d0.

Thanks @pizzathief, @WarrenWeckesser

**Comment by ev-br** [comment]
@pizzathief in case you're wondering, 'squashed and merged' message above means that your authorship is preserved, git logs are going to show you having authored 71c0f0e50e3dd8d048c7366e26ffbaa3fc137222.

For future reference: (i) Github does not send notifications on commits, (ii) some symbols have special meaning for the Sphinx doc builder (e.g. an asterisk), and they need to be wrapped into double backticks (which indicate *code* as opposed to *a text with markup*).

**Pull request closed without merging**